python_modules/dagster/dagster/_core/definitions/definitions_class.py

```diff
@@ -326,7 +328,7 @@ class BindResourcesToJobs(list):
     """
 
 
-class Definitions:
+class Definitions(DagsterModel):
```

Comment:
I punted on thinking hard about whether this should be a `DagsterModel` or a `NamedTuple`. Don't have a strong opinion here.